src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const employees = document.querySelector('ul');
+
+const listEmployees = [...employees.children];
+
+const sorted = listEmployees.sort((a, b) => {
+  return sallary(a) - sallary(b);
+});
+
+function sallary(value) {
+  const noDollar = value.dataset.salary.split('$').join('');
+  const salary = noDollar.split(',').join('');
```

Comment:
```suggestion
  const formattedSalary = value.dataset.salary.replace(/[^\d]/g, '');
```

Reply:
it's not nessesary to changed for "replace" here, but it's simpify your code, but it's nesessary to give more meanfully naming for variables, because "noDollar" little bit strange